content/blog/rails-dynamic-translation-constants.md

```diff
@@ -0,0 +1,156 @@
++++
+title = "Dynamic Translations in Rails: Why Constants Can’t Adapt to Locale Changes and How to Fix It?"
+slug = "rails-i18n"
+date = 2024-11-04T10:35:08+05:30
+image = "/images/2023/monkeypatch.png"
```

Comment:
looks like `monkeypatch.png` is a generic image which can use used in all ruby blogs
lets rename `monkeypatch.png` to `ruby-code.png` something similar and update the references